<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="flask_tests.AppTestCase-20220307130415" tests="4" file="flask_tests.py" time="0.008" timestamp="2022-03-07T13:04:17" failures="0" errors="1" skipped="0">
	<testcase classname="flask_tests.AppTestCase" name="test_account_settings_full_data" time="0.003" timestamp="2022-03-07T13:04:17"/>
	<testcase classname="flask_tests.AppTestCase" name="test_account_settings_missing_email" time="0.002" timestamp="2022-03-07T13:04:17"/>
	<testcase classname="flask_tests.AppTestCase" name="test_account_settings_missing_user" time="0.001" timestamp="2022-03-07T13:04:17"/>
	<testcase classname="flask_tests.AppTestCase" name="test_check_urls" time="0.002" timestamp="2022-03-07T13:04:17">
		<error type="RuntimeError" message="Application was not able to create a URL adapter for request independent URL generation. You might be able to fix this by setting the SERVER_NAME config variable."><![CDATA[Traceback (most recent call last):
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 59, in testPartExecutor
    yield
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "C:\Users\Johnny\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 549, in _callTestMethod
    method()
  File "c:\users\johnny\desktop\everything\school\cs 4800 software engineering\bronco-mates\src\unittest\python\flask_tests.py", line 60, in test_check_urls
    response = self.client.get(url_for("homePage"))
  File "c:\users\johnny\desktop\everything\school\cs 4800 software engineering\bronco-mates\target\venv\build\cpython-3.10.2.final.0\lib\site-packages\flask\helpers.py", line 300, in url_for
    raise RuntimeError(
RuntimeError: Application was not able to create a URL adapter for request independent URL generation. You might be able to fix this by setting the SERVER_NAME config variable.
]]></error>
	</testcase>
</testsuite>
